docs/DeveloperGuide.adoc

```diff
@@ -122,27 +122,26 @@ image::ModelClassDiagram.png[]
 The `Model`,
```

Comment:
Do you think that the multiplicity should be labelled for Task? For example, how many Drivers should there be for a single Task. 

Also, do you think that the multiplicity from ModelManager to Person should be * instead of 1? The label "filtered list" might be inaccurate for a diagram. Do you think that the label "filtered list" could be changed to a role instead? For example, "<| filteredPersonsList". 

ModelManager also has a FilteredList of Drivers and Customers. Do you think that these could be added into your diagram as well?